WooCommerce/Classes/ViewRelated/Products/Edit Product/EditableProductModel.swift

```diff
@@ -127,6 +127,18 @@ extension EditableProductModel: ProductFormDataModel, TaxClassRequestable {
         product.soldIndividually
     }
 
+    var downloads: [ProductDownload] {
+        product.downloads
+    }
+
+    var downloadLimit: Int64 {
+        Int64(product.downloadLimit)
```

Comment:
Probably it's not needed to cast it, we should update the original type.

Reply:
Here it's actually needed. Because `product.downloadLimit` in an `Int`. if I remove the casting, It throws a compilation error.

Reply:
You should update the `ProductFormDataModel` because everywhere these properties are declared as `Int`, just in the `ProductFormDataModel` protocol are declared as `Int64`.

Reply:
I am afraid, it's just quite opposite. I can see everywhere these properties are declared as `Int64`, other than just a few handfuls of places.
In Networking models, in Storage/CodeData models, these properties have been declared as `Int64` 
Please suggest. And rectify me if I miss anything.

Reply:
Ohh in that case, it's ok to use the type `Int64` everywhere instead of `Int`

Reply:
There is one issue to do that though.
In `ProductVariation`  model and in all code data models, those 2 properties (`downloadLimit` and `downloadExpiry`) are declared as Int64.
But the only place where it’s declared as `Int`, is `Product.swift`
That’s why I needed to convert to Int64 in this 2 places.

Now If we want to use `Int64` in all places, then we need to change the type from `Int` to `Int64` in `Product.swift`and few other place, including `Models+Copiable.generated.swift` file at line 33 and 34. 
If you suggest me to change at those places, then I have just one question. Since `Models+Copiable.generated.swift` file is auto generated. How to change it's code?